tests/cluster/delta/deltaspec/tables.py

```diff
@@ -0,0 +1,41 @@
+from spetlr.deltaspec.DeltaTableSpec import DeltaTableSpec
+
+# Note that the table propery delta.columnMapping.maxColumnId is completely ignored
+
+base = DeltaTableSpec.from_sql(
+    """
+    CREATE TABLE myDeltaTableSpecTestDb{ID}.tbl
+    (
+        c double,
+        d string NOT NULL COMMENT "Whatsupp",
+        onlyb int,
+        a int,
+        b string
```

Comment:
maybe also test with a column that are the same position and has same datatype etc = a total non changed column

Reply:
actually if you read the local unittest, not every column is changed.